plugins/modules/docker_network.py

```diff
@@ -35,6 +35,17 @@
     aliases:
       - network_name
 
+  config_from:
+    description:
+      - Specifies the config only network to use the config from.
+    type: str
+
+  config_only:
+    description:
+      - Sets that this is a config only network.
+    type: bool
+    default: false
```

Comment:
There should be no `default` to avoid unnecessarily changing network configs that were not created/adjusted with this module.
```suggestion
    version_added: 3.10.0
```